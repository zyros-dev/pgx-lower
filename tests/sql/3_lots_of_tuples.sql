LOAD 'pgx_lower.so';
SET client_min_messages TO WARNING;

DROP TABLE IF EXISTS test;
CREATE TABLE test(id SERIAL);

-- Insert values 1 through 5000
INSERT INTO test(id)
SELECT generate_series(1, 1000000);

SELECT COUNT(*) FROM test;
