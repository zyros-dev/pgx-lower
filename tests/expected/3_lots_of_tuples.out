LOAD 'pgx_lower.so';
NOTICE:  Installing custom executor hook...
NOTICE:  Registering custom sigsegv handler!
SET client_min_messages TO WARNING;
DROP TABLE IF EXISTS test;
CREATE TABLE test(id SERIAL);
-- Insert values 1 through 5000
INSERT INTO test(id)
SELECT generate_series(1, 1000000);
SELECT COUNT(*) FROM test;
ERROR:  Agg not yet implemented
CONTEXT:  parallel worker
